src/Model/DAOs/LdapUserDao.php

```diff
@@ -56,17 +56,19 @@ public function __construct(Ldap $ldap, LdapPasswordService $ldapPasswordService
      */
     public function getUserByCredentials($login, $password)
     {
-        $user = $this->getUserByLogin($login);
-        if ($user) {
-            $sha1Password = $this->ldapPasswordService->convertToLdapSha1($password);
```

Comment:
I think the ldapPasswordService is not needed anymore.
You could simply remove it from the package.